### DISCUSSION TITLE
Cheat List Helper

### DISCUSSION THREAD

**Discussion by a-r-db** [DarkWav/SAC-rewrite#4] [Ideas]
I found a list of cheats that are currently being used by some gamer friends of mine.

Here's an image.

![Hack List](https://user-images.githubusercontent.com/8563086/84184432-53a09480-aa5b-11ea-85d4-41c01d0c6b84.jpg)

Are any of theses cheats blockable and if so what can I do to help?

Thanks,
Austin

P.S. A list of the cheat I have blocked already.

\\\\\\
Needed
//////

Anti - XRay
Anti - NoKnockBack
Anti - Chest Esp
Anti - NoFall
Anti - Tower
Anti - Scaffold
Anti - Fast Ladder
Anti - KillAura
Anti - Horion
Anti - Fucker
Anti - Speed
more...

\\\\\\\\\\\\\\\\\\
Have AntiCheat For
//////////////////

InstaBreak
FlightA (Teleporter) (AirJump)
AutoClicker
Nuke
NoClip (Phase)

**Comment by radondev** [comment]
That's just Horion. I am sure DarkWav knows about most if not all of them.

If you are looking towards using this anticheat, I suggest taking a look at this [issue/answer in particular](https://github.com/DarkWav/SAC/issues/98#issuecomment-636503434).

> The current version (SAC v3) is not actively maintained any more.
I am working on porting the code to a new architecture, but no version is considered fully stable.
If you want a powerful production-ready anticheat, I'd recommend Mavoric v1 as mentioned.
Mavoric v2 and SAC v4 are good in theory but in an unfinished state.

**Comment by DarkWav** [comment]
Anti-Autosoup would require some advanced heuristics to detect as distinguishing between Hotkey-using Windows 10 players and actual hacked clients using RandomNumberGenerator (RNG)-based soup timings isn't exactly an easy task. Sure, some poorly coded clients could be blocked by measuring the soup timing consistency and just plain blocking instant souping (souping directly after hitting someone), but  unfortunately if the client developer puts in the necessary effort autosoup is almost undetectable.

**Comment by DarkWav** [comment]
I converted this into a discussion as it is more of a discussion than an issue, I hope you are fine with this.
